packages/slice-machine/components/AppLayout/Navigation/Icons/SliceMachineLogo.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+interface SliceMachineLogoProps {
+  fill?: string;
+  width?: string;
+  height?: string;
+}
+const SliceMachineLogo: React.FC<SliceMachineLogoProps> = ({
+  fill,
+  width,
+  height,
+}) => (
+  <svg
```

Comment:
Hmmm i've never seen an svg created that way.

Reply:
Some project use custom Webpack SVG loader, to avoid creating react component for every SVG. As this project use already this technique I used it.